### DISCUSSION TITLE
How to  use variables?

### DISCUSSION THREAD

**Discussion by binsarjr** [scalar/scalar#1552] [Q&A]
Does anyone know how to use the variable? I'm frustrated that I can't find it in the documentation at all.

**Comment by marclave** [comment]
hi @binsarjr are you trying to use the CSS Variables? or which variables are you struggling to use?

**Reply by binsarjr** [reply]
<img width="601" alt="Screenshot 2024-04-29 at 14 42 50" src="https://github.com/scalar/scalar/assets/33781288/b5ceb9a6-2e7d-49ad-8b6a-0c1baab1117a">


this one. how to use it in query string or body?

**Reply by marclave** [reply]
those are generated from the openapi specification,and get sent along with test requests.

how are you trying to use the variables? do you want to add more? :)

**Reply by binsarjr** [reply]
ah I see. I guess this is similar to insomnia that can be set directly from here. thank you

**Reply by hanspagel** [reply]
You can use them like this: {{ variableName }}

We need to make this more clear
